src/layers/medPython/cmake/add_python_module.cmake

```diff
@@ -0,0 +1,169 @@
+################################################################################
+#
+# medInria
+#
+# Copyright (c) INRIA 2021. All rights reserved.
+#
+# See LICENSE.txt for details in the root of the sources or:
+# https://github.com/medInria/medInria-public/blob/master/LICENSE.txt
+#
+# This software is distributed WITHOUT ANY WARRANTY; without even
+# the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE.
+#
+################################################################################
+
+function(add_python_module target_name)
```

Comment:
it could be interesting to add explanations in comments in this file for future readers